res/img/element-icons/room/composer/plain_text.svg

```diff
@@ -4,31 +4,14 @@
    height="20"
    viewBox="0 0 20 20"
    fill="none"
-   xmlns="http://www.w3.org/2000/svg"
-   >
-  <g
-     clip-path="url(#clip0_1456_146365)"
-     id="g53">
-    <path
-       d="M7.00042 13.7333H13.6671L14.5471 15.8667C14.7471 16.3467 15.2137 16.6667 15.7337 16.6667C16.6537 16.6667 17.2671 15.72 16.9071 14.88L11.7337 2.92C11.4937 2.36 10.9471 2 10.3337 2C9.72042 2 9.17375 2.36 8.93375 2.92L3.76042 14.88C3.40042 15.72 4.02708 16.6667 4.94708 16.6667C5.46708 16.6667 5.93375 16.3467 6.13375 15.8667L7.00042 13.7333ZM10.3337 4.64L12.8271 11.3333H7.84042L10.3337 4.64Z"
-       fill="#C1C6CD"
-       id="path49" />
-    <path
-       d="m 1.497495,8.96927 c 0,0.793654 0.7402877,1.441437 1.6473569,1.441437 H 17.521786 c 0.907096,0 1.647419,-0.647783 1.647419,-1.441437 0,-0.7936857 -0.740323,-1.4414375 -1.647419,-1.4414375 H 11.127487 3.1448519 c -0.4734211,0 -0.9014103,0.1764504 -1.2024293,0.4580061 C 1.7722258,8.1450309 1.6426187,8.3378225 1.568339,8.5513189 1.522281,8.6837006 1.497495,8.8240421 1.497495,8.96927 Z"
-       fill="#c1c6cd"
-       stroke="#ffffff"
-       id="path51"
-       style="stroke:none;stroke-width:0.840525;stroke-opacity:1" />
-  </g>
-  <defs
-     id="defs58">
-    <clipPath
-       id="clip0_1456_146365">
-      <rect
-         width="20"
-         height="20"
-         fill="white"
-         id="rect55" />
-    </clipPath>
-  </defs>
+   xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_2168_154906)">
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M14.5467 15.8667L13.6667 13.7333H7.00005L6.13338 15.8667C5.93338 16.3467 5.46672 16.6667 4.94672 16.6667C4.02672 16.6667 3.40005 15.72 3.76005 14.88L5.00323 12.0059H15.6635L16.9067 14.88C17.2667 15.72 16.6534 16.6667 15.7334 16.6667C15.2134 16.6667 14.7467 16.3467 14.5467 15.8667ZM13.6435 7.33594L11.7334 2.92C11.4934 2.36 10.9467 2 10.3334 2C9.72005 2 9.17338 2.36 8.93338 2.92L7.02326 7.33594H9.32912L10.3334 4.64L11.3376 7.33594H13.6435Z" fill="#8D97A5"/>
```

Comment:
New icons should use fill="currentColor" to allow easy styling

Reply:
\+ it would really help to use a version without padding included. Because it may change depending on the actual site of the image in the application. Design always wants a 1px around icons.

Reply:
Yeah I agree but today in usage, it's different.

If we want to have this paradigm applied to all the icons of the bottom of the room (attachment, three dots etc), we will need to have new icons for all of them. To avoid custom styling for each icon.
And this is not the purpose of this PR (only swap one icon), not to redesign all the icons of the `MessageComposer`.